### DISCUSSION TITLE
Is it possible to use Fluent UI 8 based components with Fluent UI 9?

### DISCUSSION THREAD

**Discussion by kroudams** [microsoft/fluentui#25485] [Q&A] [answered]
Is it possible to use Fluent UI 8 based components with Fluent UI 9?
For example, if I already have some utility components (e.g. Input, EmptyState, etc...) and I want to use them in my Fluent UI 9 based component implementation.

**Comment by khmakoto** [comment, marked as answer]
Covered in #25671
